src/uikit.cs

```diff
@@ -4914,6 +4936,11 @@ public partial interface UIFontDescriptor : NSSecureCoding, NSCopying {
 		[Static, Export ("preferredFontDescriptorWithTextStyle:")]
 		UIFontDescriptor GetPreferredDescriptorForTextStyle (NSString uiFontTextStyle);
 	
+		[iOS (10,0)]
+		[Static]
+		[Export ("preferredFontDescriptorWithTextStyle:compatibleWithTraitCollection:")]
+		UIFontDescriptor GetPreferredFontDescriptorForTextStyle (NSString uiFontTextStyle, [NullAllowed] UITraitCollection traitCollection);
```

Comment:
The name is inconsistent with the previous/older API name (which dropped the `Font` part) just above.